src/styles.scss

```diff
@@ -2,3 +2,7 @@
 
 /* Importing Bootstrap SCSS file. */
 @import '~bootstrap/scss/bootstrap';
+
+body {
+    padding: 0 !important;
```

Comment:
Why do you need this one?

Reply:
I don't know if it came from angular or boostrap default configuration, but the app was with a default padding, so it wasn't looking good with that style.